client/pages/projects/index.jsx

```diff
@@ -280,6 +281,12 @@ export default function ProjectsPage() {
                     >
                       <a>
                         <Card className="project-card">
+                          <CardImg
```

Comment:
And this one please, don't know why these past changes are reappearing on your branch